ui/components/Modals/ImportModal.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { useEffect } from 'react';
+import { Button, Grid } from '@material-ui/core';
+// import { createTheme } from '@material-ui/core/styles';
+import validator from "@rjsf/validator-ajv8";
+import {
+  Dialog, DialogActions,
+  DialogContent,
+  DialogTitle
+} from '@material-ui/core';
+import { Form } from '@rjsf/material-ui';
+import useStyles from "../MesheryPatterns/Cards.styles";
+import PublicIcon from '@material-ui/icons/Public';
+import filterSchema from "../../assets/jsonschema/filterImport.json"
+
+const schemaMap = {
+  filter: filterSchema,
+  application: {},
+  design: {}
+}
+
+export default function ImportModal(props) {
```

Comment:
You can use the Component mentioned below, to ease the work around.

https://github.com/meshery/meshery/blob/master/ui/components/Modal.js

Reply:
Or, you can bend the component to achieve what you need.

Reply:
@harkiratsm I think the changes I made in Modal.js can be helpful here so that we don't need to create this separate modal